lib/rss/rss.rb

```diff
@@ -593,7 +593,7 @@ def itunes_episode_type_writer(name, disp_name=name)
       module_eval(<<-DEF, *get_file_and_line_from_caller(2))
         def #{name}=(new_value)
           if @do_validate and
-               !["Full", "Trailer", "Bonus", nil].include?(new_value)
+               !["full", "trailer", "bonus", nil].include?(new_value.downcase)
```

Comment:
Could you accept only `Full`/`full` variants? (We don't want to accept `FULL`, `fULL` and so on.

```suggestion
               !["Full", "full", "Trailer", "trailer", "Bonus", "bonus", nil].include?(new_value)
```